### ISSUE TITLE
Do you have any suggestions for network design？

### ISSUE THREAD

**Issue by sneffort** [entron/entity-embedding-rossmann#12]
Thanks for your paper. And I designed a network like your paper.But I have about 600 numerical features and 10 category features. So the numerical features use Dense(1, input_dim=1) and category feature use Embedding().But when i train, the loss is nan....

**Comment by entron** [comment]
It is comman to have the nan issue when training neural network. Maybe this can help? https://stackoverflow.com/questions/33962226/common-causes-of-nans-during-training
